pyanaconda/progress.py

```diff
@@ -49,8 +50,11 @@ def progress_report(message):
     yield
     progress_step(message)
 
-def progress_message(message):
-    progressQ.send_message(message)
+def progress_message(message, xlate=True):
```

Comment:
Check the use of this in pyanaconda/install.py (I'm not totally sure but I think xlate=False is needed for that call) and in pyanaconda/ui/lib/entropy.py where it already passes in a translated message.

There are also a number of calls to progress_report passing in translated strings so maybe progress_report should also set xlate=False.